iscsi/targetd/cmd/start.go

```diff
@@ -30,55 +29,52 @@ import (
 	"k8s.io/client-go/tools/clientcmd"
 )
 
-var log = logrus.New()
+//var log = logrus.New()
 
 // start-controllerCmd represents the start-controller command
 var startcontrollerCmd = &cobra.Command{
 	Use:   "start",
 	Short: "Start an iscsi dynamic provisioner",
 	Long:  `Start an iscsi dynamic provisioner`,
 	Run: func(cmd *cobra.Command, args []string) {
-		initLog()
-		log.Debugln("start called")
+		glog.V(2).Infoln("start called")
 		var config *rest.Config
 		var err error
 		master := viper.GetString("master")
 		kubeconfig := viper.GetString("kubeconfig")
 		// creates the in-cluster config
-		log.Debugln("creating in cluster default kube client config")
+		glog.V(2).Infoln("creating in cluster default kube client config")
 		if master != "" || kubeconfig != "" {
 			config, err = clientcmd.BuildConfigFromFlags(master, kubeconfig)
 		} else {
 			config, err = rest.InClusterConfig()
 		}
 		if err != nil {
-			log.Fatalln(err)
+			glog.Fatalln(err)
 		}
-		log.WithFields(logrus.Fields{
-			"config-host": config.Host,
-		}).Debugln("kube client config created")
+		glog.V(2).Infoln("kube client config created: ", config.Host)
```

Comment:
When you write `glog.Infoln("something: ", val)`, you don't need space after `:`. Please refer to below output:

~~~
$ go run a.go
E1211 11:03:18.638285   15510 a.go:10] test:  test
E1211 11:03:18.638482   15510 a.go:11] test: test
~~~

~~~
package main

import (
	"flag"

	"github.com/golang/glog"
)

func main() {
	flag.Parse()
	glog.Errorln("test: ", "test")
	glog.Errorln("test:", "test")
}
~~~